src/Jupyter/KernelProperties.cs

```diff
@@ -18,7 +22,7 @@ public static class Constants
             DisplayName = "Q#",
 
             LanguageName = "qsharp",
-            LanguageVersion = "0.4",
+            LanguageVersion = "0.10",
```

Comment:
Will this change update the `"version": "0.4"` field in the notebooks the next time they are edited and saved? Not objecting, just want to be prepared :-)

Reply:
I believe so, yes. Ideally, we'd keep this in sync so that the metadata in notebooks is accurate, but as you can tell we haven't been great about that...